### ISSUE TITLE
Research channel backup compatibility with lnd, Umbrel etc

### ISSUE THREAD

**Issue by hsjoberg** [hsjoberg/blixt-wallet#339]

**Comment by Darth-Coin** [comment]
OK will share my test results here (having an Umbrel node ready).
So far I did this:
- fresh install latest vers Blixt, on Android 8.1
- used Umbrel aezeed seed to recover the wallet
- I couldn't see the latest txs and balance (I have some sats in onchain) due to still syncing the blocks
- I went to Settings - show node info and it displays my Umbrel node ID. Also it displays block height 689435 (latest block), but a bit down it says synced to chain - false
- wen to see the lnd log (last option in settings)and the last lines are:
LTND: Waiting for chain backend to finish sync, start_height=689435
BTCN: Registering block subscription id=2
LNML: started rescan from block ..... (height 672559) for 12 addresses
BTCN: Cancelling block subscription id=2. Before that line

And it stays like that for hours.
Tried force close and restart and the log is like stalled in that section.
